oauth2_http/javatests/com/google/auth/oauth2/AwsCredentialsTest.java

```diff
@@ -293,7 +297,7 @@ public void createdScoped_clonedCredentialWithAddedScopes() {
     AwsCredentials credentials =
         (AwsCredentials)
             AwsCredentials.newBuilder(AWS_CREDENTIAL)
-                .setServiceAccountImpersonationUrl("tokenInfoUrl")
+                .setServiceAccountImpersonationUrl(SERVICE_ACCOUNT_IMPERSONATION_URL)
```

Comment:
It feels weird to be providing the full impersonation URL vs the `targetPrinciple` (it looks like we're just parsing it out of the URL). Is this full URL being provided by some credential file?

Reply:
Yes - the URL is being provided by a credential file that will be generated by gcloud.